ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -341,10 +341,15 @@ + (ORKDateAnswerFormat *)dateAnswerFormatWithDefaultDate:(NSDate *)defaultDate
 + (ORKTextAnswerFormat *)textAnswerFormat {
     return [ORKTextAnswerFormat new];
 }
+
 + (ORKTextAnswerFormat *)textAnswerFormatWithMaximumLength:(NSInteger)maximumLength {
     return [[ORKTextAnswerFormat alloc] initWithMaximumLength:maximumLength];
 }
 
++ (ORKTextAnswerFormat *)textAnswerFormatWithValidationExpression:(NSString *)expression validInputDescription:(NSString *)description {
+    return [[ORKTextAnswerFormat alloc] initWithValidationExpression:expression validInputDescription:description];
+}
+
 + (ORKEmailAnswerFormat *)emailAnswerFormat {
     return [ORKEmailAnswerFormat new];
```

Comment:
Can you update the `ORKEmailAnswerFormat` to use an `ORKTextAnswerFormat` with `EmailValidationRegex` located at the top of this file.